docs/advanced/developer-guide.md

```diff
@@ -54,32 +54,19 @@ attribute in the spec template(s) to the new location
 
 ### Deployment
 
-The `yamls` makefile generates deployment specs matching your locally built
-image. See [build customization](#customizing-the-build) below for
-configurability, e.g. changing the deployment namespace.
+The `yamls` makefile generates a `kustomization.yaml` matching your locally
+built image and using the `deploy/overlays/default` deployment. See
+[build customization](#customizing-the-build) below for configurability, e.g.
+changing the deployment namespace.
 
 ```bash
 K8S_NAMESPACE=my-ns make yamls
-kubectl apply -f nfd-master.yaml
-kubectl apply -f nfd-worker-daemonset.yaml
+kubectl apply -k kustomize.yaml
```

Comment:
this seems to have issues.

after `kubectl apply -k .` (should be . instead of kustomize.yaml) I can see the rbac rules are still created in `node-feature-discovery` ns.

Reply:
Really good catch! This is now fixed in a separate patch on top of the old ones.